btr-api/src/btr_api/services/auth.py

```diff
@@ -66,59 +72,129 @@ def init_app(self, app: Flask):
+                self.app.logger.info('Get User Type: ' + self.USER_STAFF)
+                return self.USER_STAFF
+        except Exception as e:
+            self.app.logger.error('Error in Get User Type: ' + str(e))
+
+        self.app.logger.info('Get User Type: ' + self.USER_PUBLIC)
+        return self.USER_PUBLIC
+
     def get_bearer_token(self):
         """Get a valid Bearer token for the service to use."""
         data = 'grant_type=client_credentials'
         try:
-            res = requests.post(url=self.sso_svc_token_url,
-                                data=data,
-                                headers={'content-type': 'application/x-www-form-urlencoded'},
-                                auth=(self.svc_acc_id, self.svc_acc_secret),
-                                timeout=self.sso_svc_timeout)
+            res = requests.post(
+                url=self.sso_svc_token_url,
+                data=data,
+                headers={'content-type': 'application/x-www-form-urlencoded'},
+                auth=(self.svc_acc_id, self.svc_acc_secret),
+                timeout=self.sso_svc_timeout,
+            )
             if res.status_code != HTTPStatus.OK:
                 raise ConnectionError({'statusCode': res.status_code, 'json': res.json()})
             return res.json().get('access_token')
         except exceptions.Timeout as err:
             self.app.logger.debug('SSO SVC connection timeout: %s', err.with_traceback(None))
-            raise ExternalServiceException(HTTPStatus.GATEWAY_TIMEOUT,
-                                           [{'message': 'Unable to get service account token.',
-                                             'reason': err.with_traceback(None)}]) from err
+            raise ExternalServiceException(
+                HTTPStatus.GATEWAY_TIMEOUT,
+                [{'message': 'Unable to get service account token.', 'reason': err.with_traceback(None)}],
+            ) from err
         except Exception as err:  # noqa: B902
             self.app.logger.debug('SSO SVC connection failure: %s', err.with_traceback(None))
-            raise ExternalServiceException(HTTPStatus.SERVICE_UNAVAILABLE,
-                                           [{'message': 'Unable to get service account token.',
-                                             'reason': err.with_traceback(None)}]) from err
+            raise ExternalServiceException(
+                HTTPStatus.SERVICE_UNAVAILABLE,
+                [{'message': 'Unable to get service account token.', 'reason': err.with_traceback(None)}],
+            ) from err
 
     def get_authorization_header(self, request) -> str:
         """Gets authorization header from request."""
         authorization_header = request.headers.get('Authorization', None)
         if not authorization_header:
-            error = f'Missing authorization header: {request.headers}'
+            error = f"Missing authorization header: {request.headers}"
```

Comment:
styling: keeping the quotes same through out the code

Reply:
This is resolved